model-optimizer/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -161,7 +161,7 @@ def xml_ports(node: Node, element: Element, edges: Element):
                 outputs = SubElement(element, 'output')
             port = SubElement(outputs, 'port')
             port.set('id', str(d['out']))
-            port_id = d['out'] - len(node.in_nodes())
+            port_id = d['out'] - len(node.in_nodes()) if node.type != 'Const' else d['out']
```

Comment:
This solution works but it does not look beautiful. Can we consider refactoring of Const op IR generation so there will be no need in this check here?

Reply:
To be honest I don't know how to make this part better. I have tried some variants, but they are looks bad.

Reply:
The transformation `CreateConstNodesReplacement` from file `model-optimizer/extensions/back/SpecialNodesFinalization.py` adds a copy of the data node in lines 104-106 as input to the `Const` op node. This data node was added to properly generate the Const operation long time ago (this data node is the reason why the Const output port start from 1 and not 0). I would recommend to not add this redundant data node and modify the emitter code to correctly work for this case.

Reply:
Never mind. Please, ignore this comment.